scripts/setup_ubuntu.sh

```diff
@@ -110,10 +110,15 @@ apt update
 # Install packages required by the requirements tool
 apt install --yes --quiet python3-yaml python3-packaging
 
+if [ "$VERSION" == "18" ]; then
+  PKG_SYS_SUFFIX="-18.04"
+else
+  PKG_SYS_SUFFIX=""
+fi
 # Use the requirements tool to install apt packages with: apt-get satisfy --allow-downgrades --yes --quiet
-"$REPO_ROOT"/scripts/requirements install -a--allow-downgrades -a--yes -a--quiet -l apt -l apt/dev -f apt
+"$REPO_ROOT"/scripts/requirements install --arg=--allow-downgrades --arg=--yes --arg=--quiet --label apt --label apt/dev --packaging-system "apt$PKG_SYS_SUFFIX"
```

Comment:
I have to admit, I didn't know the `-a--allow-downgrades` syntax and I found it confusing.  I prefer the explicit `--arg`.

Reply:
Yea. `-a` was  a bad idea in a script. This reminds me of my own advice, never use contractions in code without a really good reason and good documentation. I should listen to me. ;-)